### PR TITLE
Implemented batch wait timer

### PR DESCRIPTION
**Pull request by DmitryKuzmenko**
This is for #1038 

Please check my "batch-wait" cmdline arg description. Is it clear enough and correct?

### CODE REVIEW

File: salt/utils/parsers.py [hunk 1]
```diff
@@ -1831,6 +1831,14 @@ def _mixin_setup(self):
                   'minions to have running')
         )
         self.add_option(
+            '--batch-wait',
+            default=0,
+            dest='batch_wait',
+            type=float,
+            help=('Wait the specified time in seconds after each job is done '
+                  'before free the slot in the batch for the next one')
```
Comment (cachedout) [inline salt/utils/parsers.py]: `before freeing`

**Comment by cachedout** [comment]
Hi @DmitryKuzmenko . Just a tiny english grammar fixup and we'll be ready for this to go in. Thanks!

**Comment by cachedout** [comment]
@DmitryKuzmenko I just went ahead and fixed this so no need to worry about it. Thanks.

**Comment by DmitryKuzmenko** [comment]
Thank you @cachedout !

**Pull request merged**